src/shell/pipe_exec/command_not_found.rs

```diff
@@ -0,0 +1,35 @@
+use super::super::{Capture, Function, Shell};
+use parser::shell_expand::expand_string;
+use std::io::Read;
+use std::process;
+use sys;
+
+pub(crate) fn command_not_found(shell: &mut Shell, command: &str) -> Option<String> {
+    let function = match shell.functions.get("COMMAND_NOT_FOUND") {
+        Some(func) => func as *const Function,
+        None => return None // TODO: Use ? on Option whenever we drop support for older rust versions
+    };
+
+    let mut output = None;
```

Comment:
Is capturing output only to print it out later even necessary? I don't know, you tell me.

Reply:
It is not necessary. You don't have to capture anything here.

Reply:
Hmm, it doesn't print anything when I change it to `Capture::None` and remove all capture code